pyomo/gdp/bigm.py

```diff
@@ -9,18 +9,19 @@
 #  ___________________________________________________________________________
 
 from pyomo.common.deprecation import deprecated
-from pyomo.common.plugin import Plugin, implements
-from pyomo.core import IPyomoScriptModifyInstance, TransformationFactory
+from pyomo.core import TransformationFactory
 
 # This import ensures that gdp.bigm is registered, even if pyomo.environ
 # was never imported.
 import pyomo.gdp.plugins.bigm
 
+#
+# TODO - Should we keep this plugin?
+#
+@TransformationFactory.register("gdp._bigm")
```

Comment:
So ... what changes are needed? I'm not sure what to do with this plugin.

Reply:
This is a plugin for the Pyomo Workflow (NOT the TransformationFactory).  If you are deleting the Pyomo Workflow (another break in backwards compatibility), then this can be deleted.  If you are migrating the Pyomo Workflow over to the new system, then these need to be migrated as well.  If you are leaving it alone, then these should be left alone.